content/edu/chicago.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: R-Ladies Chicago Package Development Workshop
+---
+
+## Women's Package Development Workshop by [R-Ladies Chicago](https://rladieschicago.org) and [FoRwards](https://forwards.github.io)
+
+### Date: Saturday, February 23, 2018, 9:30am-4pm
+
+### Location
+
+Center for Spatial Data Science  
+5735 S Ellis Ave, 2nd Floor  
+Chicago, Illinois 60637 
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2974.7548639089937!2d-87.60317058411151!3d41.79049637886998!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x880e293e7bd41241%3A0x5966b3036fc25171!2sSearle+Chemical+Laboratory%2C+5735+S+Ellis+Ave%2C+Chicago%2C+IL+60637!5e0!3m2!1sen!2sus!4v1544044603496" width="600" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+ 
+### Applications
+
+Applications for this event will be opened in **late December 2018**. Please [join our Meetup](https://www.meetup.com/rladies-chicago/) for updates, and stay tuned for more!
+
+### Workshop Description
+
+Curious about package development, but not sure where to start? This workshop is for you! Run by R-Ladies Chicago, the goal of this workshop is to empower you to start contributing back to the R ecosystem with package development. We'll be using materials developed by Hadley Wickham, Jenny Bryan, and Di Cook for the [FoRwards organization](https://forwards.github.io/about/).
+
+By the end of this day-long workshop, you should know how to:
+
+- make your code into an R package, 
+- do collaborative coding with GitHub, 
+- write a vignette or an article, 
+- build a web page for your package, and
+- submit a package to CRAN. 
+
+Bring your own code that you want to make into a package, or work with our example. 
+
+### Cost and Travel Support
+Thanks to funding from the <a href="https://www.r-consortium.org/" target="_blank">R Consortium</a>, this workshop is free for participants.  Lunch will be provided.  
+
+### Lead Instructors
+
+We are currently in the process of finalizing our instructors and helpers, but Angela Li of the [Center for Spatial Data Science](https://spatial.uchicago.edu) (and R-Ladies Chicago) is organizing this event. If you would like to teach or help out, please let her know!
+
+### Code of Conduct
+
+Workshop instructors and participants agree to adhere to the <a href="https://wiki.r-consortium.org/view/R_Consortium_and_the_R_Community_Code_of_Conduct" target="_blank">R Consortium and R Community Code of Conduct</a>.
+
+### Questions
+
+Please email angela [at] rladies.org with any questions, and see <a href="http://forwards.github.io/edu/" target="_blank">here</a> for more information.
```

Comment:
I'm not sure if you need to link to the Education page, but if you want to please use an informative label and explain why people might want to go there.

```suggestion
Please email angela [at] rladies.org with any questions, and see <a href="http://forwards.github.io/edu/" target="_blank">the main Education page</a> for more information about Forwards teaching initiatives.
```